react-viewer/package.json

```diff
@@ -1,43 +1,48 @@
 {
-  "name": "viewer2",
-  "version": "0.1.0",
+  "name": "viewer",
+  "version": "0.3.0",
   "private": true,
   "dependencies": {
-    "@bentley/bentleyjs-core": "^2.19.1",
-    "@bentley/extension-client": "^2.19.1",
-    "@bentley/frontend-application-insights-client": "^2.19.1",
-    "@bentley/frontend-authorization-client": "^2.19.1",
-    "@bentley/geometry-core": "^2.19.1",
-    "@bentley/icons-generic": "^1.0.31",
-    "@bentley/imodelhub-client": "^2.19.1",
-    "@bentley/imodeljs-common": "^2.19.1",
-    "@bentley/imodeljs-frontend": "^2.19.1",
-    "@bentley/imodeljs-i18n": "^2.19.1",
-    "@bentley/imodeljs-markup": "^2.19.1",
-    "@bentley/imodeljs-quantity": "^2.19.1",
-    "@bentley/itwin-client": "^2.19.1",
-    "@bentley/orbitgt-core": "^2.19.1",
-    "@bentley/presentation-common": "^2.19.1",
-    "@bentley/presentation-frontend": "^2.19.1",
-    "@bentley/product-settings-client": "^2.19.1",
-    "@bentley/rbac-client": "^2.19.1",
-    "@bentley/react-scripts": "^4.0.3",
-    "@bentley/telemetry-client": "^2.19.1",
-    "@bentley/ui-abstract": "^2.19.1",
-    "@bentley/ui-components": "^2.19.1",
-    "@bentley/ui-core": "^2.19.1",
-    "@bentley/ui-framework": "^2.19.1",
-    "@bentley/ui-ninezone": "^2.19.1",
-    "@bentley/webgl-compatibility": "^2.19.1",
-    "@itwin/web-viewer-react": "^1.1.0",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "@bentley/icons-generic": "^1.0.13",
```

Comment:
Can we remove this yet? It's been something I've been wondering about the base viewer template :)

Reply:
unfortunately not :(
a lot of icons used in core and other pkgs require this pkg to be present to deliver the webfonts needed for icons
i really want to nuke this pkg in 4.0

Reply:
that's annoying okay, I'll add it to the 4.0 list.